src/uu/touch/src/touch.rs

```diff
@@ -640,6 +640,42 @@ fn parse_date(ref_time: DateTime<Local>, s: &str) -> Result<FileTime, TouchError
     Err(TouchError::InvalidDateFormat(s.to_owned()))
 }
 
+/// Prepends 19 or 20 to the year if it is a 2 digit year
+///  
+/// GNU `touch` behavior:
+///
+/// - 68 and before is interpreted as 20xx
+/// - 69 and after is interpreted as 19xx
+fn prepend_19_or_20(s: &str) -> UResult<String> {
+    let first_two_digits: &i32 = match &s[..2].parse::<i32>() {
```

Comment:
I would use `u32` instead of `i32` because it allows you to get rid of the `first_two_digits >= 0` condition on line 659.

Reply:
Ah, right !